packages/client/components/ReflectionCard/EmojiUsersReaction.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import {EmojiUsersReaction_reactji$key} from '~/__generated__/EmojiUsersReaction_reactji.graphql'
+
+const EmojiUsersReactionParent = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  maxHeight: 300,
+  maxWidth: 280,
+  overflow: 'auto',
+  wordWrap: 'break-word',
+  overflowWrap: 'break-word',
+  whiteSpace: 'normal'
+})
+
+const LIST_FORMATTER =
+  Intl.ListFormat !== undefined
+    ? new Intl.ListFormat('en', {style: 'long', type: 'conjunction'})
+    : {format: (arr: string[]) => arr.join(', ')} // fallback for safari pre Big Sur
+
+interface Props {
+  reactjiRef: EmojiUsersReaction_reactji$key
+}
+
+const EmojiUsersReaction = ({reactjiRef}: Props) => {
+  const reactji = useFragment(
+    graphql`
+      fragment EmojiUsersReaction_reactji on Reactji {
+        id
+        users {
```

Comment:
-2 looks like the demo will need to be updated. It crashes for me as it doesn't have any reactji users

Reply:
good catch! I updated the demo but seems like it didn't like the unused fragment query. I removed it and it's all good on my end. Could you please double check? 🙏